requirements/edx/base.txt

```diff
@@ -23,7 +23,7 @@ django-celery==3.1.16
 django-config-models==0.1.3
 django-countries==4.0
 django-extensions==1.5.9
-django-filter==0.11.0
+django-filter==1.0.4
```

Comment:
This is also required for data-api only IIRC

Reply:
You are right